lib/modules/apostrophe-express/index.js

```diff
@@ -592,10 +613,13 @@ module.exports = {
       _.defaults(req.data, _.pick(req, 'baseUrl', 'baseUrlWithPrefix', 'absoluteUrl'));
     };
     
-    // Locate modules with middleware and add it to the list
+    // Locate modules with middleware and add it to the list.
+    // Also compile the CSRF exceptions, late, so that other
+    // modules can respond to the `csrfExceptions` event.
     self.afterInit = function() {
+      self.enableCsrf();
```

Comment:
Make sure I didn't break CSRF for ordinary stuff by moving it so late.

Reply:
Yes this is fine, the middleware itself is added early but its configuration is supplied late.